docs: add embedding page (#759)
jina-ai/finetuner: docs/get-started/pretrained.md

(pretrained-models)=
# {octicon}`rocket` Jina Embeddings

Starting with Finetuner 0.7.9,

> we introduce a suite of pre-trained text embedding models licensed under Apache 2.0.

Reviewer: ```suggestion
we have introduced a suite of pre-trained text embedding models licensed under Apache 2.0.
```

> The model has a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.

Reviewer 2: ```suggestion
The models have a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
```

> The model have a range of use cases, including information retrieval, semantic textual similarity, text reranking, and more.

Reviewer: ```suggestion
These models have a variety of use cases, including information retrieval, semantic textual similarity, text reranking, and more.
```

> The suite includes the following:

Reviewer: ```suggestion
The suite consists of the following models:
```

> - `jina-embedding-s-en-v1` **[Huggingface](jinaai/jina-embedding-s-en-v1)**: With a compact size of just 35 million parameters, the model enables lightning-fast inference while still delivering impressive performance.

Reviewer: ```suggestion
- `jina-embedding-s-en-v1` **[Huggingface](jinaai/jina-embedding-s-en-v1)**: This is a compact model with just 35 million parameters, that performs lightning-fast inference while delivering state-of-the-art performance.
```

Author: i'll apply the rest but not SOTA, because it;s not the top one

> - `jina-embedding-b-en-v1` **[Huggingface](jinaai/jina-embedding-b-en-v1)**: With a standard size of 110 million parameters, the model enables fast inference while delivering better performance than our small model.

Reviewer: ```suggestion
- `jina-embedding-b-en-v1` **[Huggingface](jinaai/jina-embedding-b-en-v1)**: This model has a size of 110 million parameters, performs fast inference and delivers better performance than our smaller model.
```

> - `jina-embedding-l-en-v1` **[Huggingface](jinaai/jina-embedding-l-en-v1)**: With a size of 330 million parameters, the model enables single-gpu inference while delivering better performance than our small and base model.

Reviewer: ```suggestion
- `jina-embedding-l-en-v1` **[Huggingface](jinaai/jina-embedding-l-en-v1)**: This is a relatively large model with a size of 330 million parameters, that performs single-gpu inference and delivers better performance than our other model.
```

## Usage

```python
import finetuner

model = finetuner.build_model('jinaai/jina-embedding-s-en-v1')
embeddings = finetuner.encode(
    model=model,
    data=['how is the weather today', 'What is the current weather like today?']
)
print(finetuner.cos_sim(embeddings[0], embeddings[1]))
```

## Training Data

> Jina embedding models is a suit of language models that has been trained using Jina AI's Linnaeus-Clean dataset.

Reviewer 2: ```suggestion
Jina embedding models is a suit of language models that have been trained using Jina AI's Linnaeus-Clean dataset.
```

> This dataset consists of 380 million pairs of sentences, which include both query-document pairs.

Reviewer: ```suggestion
This dataset consists of 380 million query-document pairs of sentences.
```

These pairs were obtained from various domains and were carefully selected through a thorough cleaning process.
The Linnaeus-Full dataset, from which the Linnaeus-Clean dataset is derived, originally contained 1.6 billion sentence pairs.

## Characteristics

> Each Jina embedding model allows encoding of up to 512 tokens,

Reviewer: ```suggestion
Each Jina embedding model can encode up to 512 tokens,
```

> with any additional tokens being truncated.

Reviewer: ```suggestion
with any further tokens being truncated.
```

> The output dimensionality varies across different models.
> Please consult the table below for more details.

Reviewer: ```suggestion
The models have different output dimensionalities, as shown in the table below:
```
